### DISCUSSION TITLE
No need to purge  NetBSD from CI

### DISCUSSION THREAD

**Discussion by deep-soft** [Slackadays/Clipboard#120] [Show and tell]
Refering to:
https://github.com/Slackadays/Clipboard/commit/ac5b9722c9dd84c630bf1b053da702dab2fd7b65
Don't know if there is someone with netbsd who can test this, but there is no need to purge NetBSD from CI.

To fix CI:
in file:
src/cb/src/externalclipboards.cpp
at line: 17
insert this:
#include <climits>

**Comment by Slackadays** [comment]
The other reason I purged NetBSD is because the Cross Platform Actions system is really unreliable with NetBSD so it often fails for no reason. However, I can make that fix and see if it helps at all :)

**Comment by Slackadays** [comment]
@deep-soft there we go again https://github.com/Slackadays/Clipboard/actions/runs/5137185542/jobs/9244947916

**Comment by deep-soft** [comment]
I don't have NetBSD and can't test Clipboard on that platform, but I tried to fix CI to run on all of those platforms.
Of course it's you're call, to keep NetBSD or not.
